src/System.Private.Xml/src/Resources/System.Private.Xml.rd.xml

```diff
@@ -15,7 +15,6 @@
         </Type>
         <Type Name="ReflectionXmlSerializationReaderHelper" Dynamic="Required All" />
       </Namespace>
-      <Namespace Name="System.Xml.Schema" Dynamic="Public" />
```

Comment:
While I don't object to the change, it's not clear why this would root anything -- it should just be adding metadata to stuff we keep anyway.

Reply:
Nevermind, got it now.